dist/node.js

```diff
@@ -33,6 +33,7 @@ class node {
   constructor (options = {}) {
     Object.assign(this, {
       network: 'testnet', /** the network to connect to. */
+      nodes: [], /** {Array} optionally manually set the nodes to connect to **/
```

Comment:
use network attribute to set optional attributes similar to the override for storage:

network: {
 'nework': 'privatenet',
 'seeds': [ ]
}